### PR TITLE
Miner updates

### PR DESCRIPTION
**Pull request by tmiland**
Updated Miners and removed disabled repos.

### CODE REVIEW

**Review by incodemines — CHANGES_REQUESTED** [review]
nvidia-win.json

You missed bminer.
 "version": "16.4.8",
"uri": "https://www.bminercontent.com/releases/bminer-lite-v16.4.8-b37ccb0-amd64.zip"

**Comment by tmiland** [comment]
> 
> 
> nvidia-win.json
> 
> You missed bminer. "version": "16.4.8", "uri": "https://www.bminercontent.com/releases/bminer-lite-v16.4.8-b37ccb0-amd64.zip"

Updated. 👍

**Comment by Speeddymon** [comment]
Cryptodredge 0.26.0 is available from the Rainbowminer repository and works well with SWARM so I'd recommend updating the URL instead of deleting the miner:
https://github.com/RainbowMiner/miner-binaries/releases/download/v0.26.0-cryptodredge/CryptoDredge_0.26.0_cuda_11.2_linux.tar.gz

Also there's a typo on amd-win.json in this pull request:
`    "uri": "https://github.com/fancyIX/sgminer-phi2-branch/releases/download/0.7.5-0/sgminer-fancyIX-win64-0.7.5.zip"`
The version line up above is 0.7.6 but the URL here has 0.7.5.

File: config/update/nvidia-linux.json [hunk 1]
```diff
@@ -231,16 +220,16 @@
     "optional": "Yes",
     "uri": "https://github.com/MaynardMiner/MM.Compiled-Miners/releases/download/v5.0/grin-miner-linux.tar.gz"
   },
-  "bminer": {
+   "bminer": {
```
Comment (Speeddymon) [inline config/update/nvidia-linux.json:223]: There was an extra space added here that shouldn't be

File: config/update/amd-win.json [hunk 2]
```diff
@@ -161,9 +161,9 @@
     "AMD2": ".\\bin\\fancyix-2\\sgminer.exe",
     "AMD3": ".\\bin\\fancyix-3\\sgminer.exe",
     "minername": "sgminer.exe",
-    "version": "0.6.0.0",
+    "version": "0.7.6",
     "optional": "No",
-    "uri": "https://github.com/fancyIX/sgminer-phi2-branch/releases/download/5.6.1.3.b6/sgminer-phi2-fancyIX-win64-0.6.0.0.zip"
+    "uri": "https://github.com/fancyIX/sgminer-phi2-branch/releases/download/0.7.5-0/sgminer-fancyIX-win64-0.7.5.zip"
```
Comment (Speeddymon) [inline config/update/amd-win.json:166]: The version here is 0.7.5 but on line 164 it's 0.7.6

**Comment by MaynardMiner** [comment]
I will add this, but not create a release- It requires me having to make changes for the auto-updater to migrate benchmarks and settings.

**Pull request merged**